aop/src/main/java/io/micronaut/aop/Intercepted.java

```diff
@@ -21,5 +21,5 @@
  * @author Graeme Rocher
  * @since 1.0
  */
-public interface Intercepted {
+public interface Intercepted extends io.micronaut.inject.proxy.Intercepted {
```

Comment:
Perhaps name the new interface something like `InterceptedBean` to avoid confusion.